pkg/net/libp2p/key_test.go

```diff
@@ -136,14 +153,17 @@ func TestNetworkPublicKeyToOperatorPublicKey(t *testing.T) {
 		t.Errorf("operator public key uses the wrong curve")
 	}
 
-	if operatorPublicKey.X.Cmp(
-		networkPublicKey.(*libp2pcrypto.Secp256k1PublicKey).X,
-	) != 0 {
+	publicKey, ok := networkPublicKey.(*libp2pcrypto.Secp256k1PublicKey)
+	if !ok {
+		t.Fatal("wrong type of public key")
+	}
```

Comment:
Let's change this in the same way as proposed above

Reply:
Done in https://github.com/keep-network/keep-core/commit/027febb4837f04ec11bc4e208c0b1e28a8da3642.